Makefile

```diff
@@ -117,7 +117,7 @@ deploy-kubemark:
 .PHONY: test
 test: ## Run tests
 	@echo -e "\033[32mTesting...\033[0m"
-	$(DOCKER_CMD) hack/ci-test.sh
+	$(DOCKER_CMD) hack/unit-tests.sh
```

Comment:
I still think we should retain the old name/fix git so it notices the rename, at the moment the diff is large because it thinks we deleted and made a new file, when really these should be the same no?

Reply:
Git/GH don't treat it as rename for some reason.
Rolled back.